sftp/1.3/modules/ROOT/edits/index.adoc

```diff
@@ -0,0 +1,40 @@
+= SFTP Connector 1.3 - Mule 4
+:page-aliases: connectors::sftp/sftp-connector.adoc
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
+
+Anypoint Connector for SFTP (SFTP Connector) provides access to files and folders on a SFTP server. SFTP connector operations manage file transfers over the SFTP (Secure File Transfer Protocol) protocol. The connector main features include:
+
+* Read files or list directory contents on demand.
+* Support for common SFTP operations such as creating directories, copying, moving, renaming, deleting files.
+* Support for locking files.
+* File matching functionality.
+* Consistent design with the File and FTP Connectors. The same set of operations is available on these connectors, and they behave similarly.
+
+=== Representation of File Attributes
+
+Mule 4 provides a message structure composed of a payload and attributes, where the data moves around. This means that the payload is a file content. The attributes provide an object that contains metadata on the payload and information such as the file’s name, path, size, timestamps, and so on. For the FTP and SFTP connectors these attributes are known as the _file attributes_.
+
+The Mule message contains the concepts of encoding and MIME type. These describe the format in which a payload is represented. The file attributes describe a file’s encoding and MIME type.
```

Comment:
Does a "message" contain "concepts" or are we saying that the message uses encoding and MIME type? Clarify.

Reply:
These three sentences together include some repetition: can you pare it down?